runtime/j9vm/j9memcategories.c

```diff
@@ -84,6 +84,9 @@ OMRMEM_CATEGORY_NO_CHILDREN("Modules", J9MEM_CATEGORY_MODULES);
 #if defined(OMR_OPT_CUDA)
 OMRMEM_CATEGORY_NO_CHILDREN("CUDA", OMRMEM_CATEGORY_CUDA);
 #endif /* OMR_OPT_CUDA */
+#if JAVA_SPEC_VERSION >= 16
+OMRMEM_CATEGORY_NO_CHILDREN("Foreign Linker API", J9MEM_CATEGORY_VM_FFI);
```

Comment:
This should be declared as a child of "VM".

Reply:
Moved `J9MEM_CATEGORY_VM_FFI` to `OMRMEM_CATEGORY_9_CHILDREN` & `OMRMEM_CATEGORY_10_CHILDREN` (newly added in OMR at https://github.com/eclipse/omr/pull/6287).